src/rails_admin/styles/base/theming.scss

```diff
@@ -42,6 +42,16 @@ body.rails_admin {
     > li > a {
       padding: ($navbar-padding-y / 2) ($grid-gutter-width / 2);
 
+      &.active {
+        background-color: map-get($theme-colors, primary);
+        color: color-contrast(map-get($theme-colors, primary));
+      }
+
+      &:hover {
+        background-color: map-get($theme-colors, primary);
+        color: color-contrast(map-get($theme-colors, primary));
+      }
+
```

Comment:
The styles here can be removed if it'd be preferable to keep with the current behavior, https://github.com/railsadminteam/rails_admin/pull/3511/commits/410654d76e7de5b5c560ca51ef7a92484b1ff75f is the main change needed to allow targeting the (active sidenav) element.